### ISSUE TITLE
Saving plist file doesn't work

### ISSUE THREAD

**Issue by maroeldawwyp** [corpnewt/ProperTree#185]
Saving plist file after editing doesn't work when Caps Lock is on, it must be turned off first then save the file.

**Comment by corpnewt** [comment]
Ha - this is an interesting one.  It seems Caps Lock changes the modifiers tkinter sees in key presses.  I'll see if I can find a way to strip that mod and re-raise the events.

-CorpNewt

**Comment by corpnewt** [comment]
I pushed some tests to a new [caps-lock-test branch](https://github.com/corpnewt/ProperTree/tree/caps-lock-test) - would you mind verifying if all the key detection (hotkeys and otherwise) still works for you there?  I only tested it on macOS so far, and it *seems* to work in my testing, but as input handling is a rather large change, I'd prefer more testing over less.

-CorpNewt

**Comment by maroeldawwyp** [comment]
OK, CorpNewt.

I will test and reply result to you ASAP.

BRB...

**Comment by corpnewt** [comment]
Sure - no rush.  I tested on Windows and found that I needed to [set up some bitmasking](https://github.com/corpnewt/ProperTree/commit/51aa39002036687d919d0cf5b3f991e6d6a6f792) to bypass issues with scroll lock and num lock - but those were pretty easy to implement.  I also employed a couple other optimizations - but it *should* work on macOS, Windows, and Linux at this point I would guess.

-CorpNewt
